products/container/api.yaml

```diff
@@ -58,6 +58,20 @@ objects:
         description: 'The location where the cluster is deployed'
         required: true
     properties:
+      - !ruby/object:Api::Type::String
+        name: 'kubectlPath'
+        description: |
+          The path that the kubectl config file will be written to.
+
+          The file will not be created if this path is unset.
+
+          Any existing file at this path will be completely overridden.
```

Comment:
...completely overwritten.

Reply:
Yep, I've been living in MM-land for too long.